module-4/backend/week_one.md

```diff
@@ -5,18 +5,43 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+
 2. What are some tools to help debug JavaScript code?
+Chrome Dev Tools (Firefox too...), console.log, and eval(pry.it). They all have their place.
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha at the least, and Chai for better asserts.
 4. What is the syntax for invoking a function?
+```javascript
+function() <- needs the parantheses, function just returns the actual function, which is useful for sending callbacks etc...
+```
 5. What is CORS?
+Cross-Origin Resource Sharing. My understanding is that it allows one web origin to access data from another web origin.
 6. How do we enable CORS?
+I have no idea. Yet.
```

Comment:
Server-side config will set this up to restrict requests to API. It's enabled automatically client-side by Chrome